source/user-manual/ruleset/testing.rst

```diff
@@ -5,12 +5,15 @@
 Testing decoders and rules
 ===============================
 
-The tool *ossec-logtest* allow us to test how an event is decoded and if an alert is generated.
+Each environment needs a specified ruleset. The Sysadmins eventually will need to create their own rules and decoders. 
 
-Run the tool */var/ossec/bin/ossec-logtest* and paste the following log::
+When creating rules and decoders there is a need to verify they work. The Logtest tool allows the testing and verification of they.
```

Comment:
There are some grammar issues. It should be "When creating rules and decoders, there is a need to verify that they work. The Logtest tool allows their testing and verification."